src/DotNetWorker.ApplicationInsights/FunctionsApplicationInsightsExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
```

Comment:
nit - Are there certain files that need the copyright header and some that don't?

Reply:
No they all should... will go through and double-chedk!